feat: docker-build action (#1)
bit4coinBV/github-actions: README.md

Example of usage with custom GITHUB Token for Release Please can be found [here](examples/jira-release.yaml).

Example of usage in single file can be found [here](examples/release.yaml).

## Docker Build

> Builds and Pushes Docker image

Reviewer: ```suggestion
Builds a docker image and pushes it to a registry
```

Author: done

> By default it will tags image with commit short-sha and tag (tag when workflow triggered from tag).

Reviewer: ```suggestion
By default, the image will be always tagged with the short-sha of the latest commit and with the tag name if the build was triggered from a GitHub tag.
```

Author: done

> You can also pass `version` parameter which has higher priority.

Reviewer: ```suggestion
You can also pass the version explicitly with the `version` parameter.
```
